patches/build-config-compiler-BUILD.gn.patch

```diff
@@ -1,7 +1,16 @@
 diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
-index bd1fefeff6f604474cbe1bafb89513b12abf8dba..d67093b26f09ce2e12edb013d21189ec69ddcf92 100644
+index bd1fefeff6f604474cbe1bafb89513b12abf8dba..d37a90ca7402c826060c8f2b4b2970e5795af4c9 100644
 --- a/build/config/compiler/BUILD.gn
 +++ b/build/config/compiler/BUILD.gn
+@@ -1269,7 +1269,7 @@ config("compiler_deterministic") {
+     } else {
+       # -ffile-compilation-dir is an alias for both -fdebug-compilation-dir=
+       # and -fcoverage-compilation-dir=.
+-      cflags += [ "-ffile-compilation-dir=." ]
++      cflags += [ "-fdebug-compilation-dir=." ]
```

Comment:
No logic change here, but goma local cache hits will start to work.

Reply:
I'm still considering if we should do this or just keep it as is. Without this change remote goma cachehits will still work, but local cachehits will not be possible across different working dirs.
The main concern is that this change will trigger full rebuild for anyone, and when we will revert this, it's gonna be another full rebuild for anyone.

Reply:
discussed with @bridiver, decided to remove this as it's not worth the temporary benefits.